src/test/java/duke/commands/ExitCommandTest.java

```diff
@@ -0,0 +1,13 @@
+package duke.commands;
```

Comment:
Maybe consider adding more tests since your code is so well-structured and thus conducive to testing